src/relayer/index.ts

```
@@ -240,13 +244,17 @@ export async function createRelayer() {
 
           if (shouldWait) {
             const beforeWait = performance.now()
-            const receipt = await relayerSmartClient
-              .waitForUserOperationReceipt({
-                hash: uoHash,
-                pollingInterval: 1000,
-                retryCount: tryNumber * 3 + 2,
-              })
-              .catch(injectUOError({ uoError: UOError.FailedWaitForOperation, account, signer }))
+            const receipt = await Promise.any(
+              allHashes.map((hash, i) =>
+                relayerSmartClient
+                  .waitForUserOperationReceipt({
+                    hash,
+                    pollingInterval: 750,
+                    retryCount: i + 3,
```

Is this an intentional change to retry count logic?

---

yeah the logic here is to "fail fast" for the initial attempt but then catch any success that may happen down the line with the allHashes logic